nautobot/wireless/tests/test_api.py

```diff
@@ -0,0 +1,279 @@
+from nautobot.core.testing import APIViewTestCases
+from nautobot.dcim.models import Controller, Device
+from nautobot.extras.models import SecretsGroup
+from nautobot.ipam.models import VLAN
+from nautobot.tenancy.models import Tenant
+from nautobot.wireless import models
+
+
+class AccessPointGroupTest(APIViewTestCases.APIViewTestCase):
+    model = models.AccessPointGroup
+
+    @classmethod
+    def setUpTestData(cls):
+        tenants = Tenant.objects.all()
+        controllers = Controller.objects.all()
+        models.AccessPointGroup.objects.create(
+            name="Access Point Group 1", controller=controllers[0], tenant=tenants[0]
+        )
+        models.AccessPointGroup.objects.create(
+            name="Access Point Group 2", controller=controllers[1], tenant=tenants[1]
+        )
+        models.AccessPointGroup.objects.create(
+            name="Access Point Group 3", controller=controllers[2], tenant=tenants[2]
+        )
+        cls.create_data = [
+            {
+                "name": "Access Point Group 4",
+                "controller": controllers[3].pk,
+                "tenant": tenants[3].pk,
+                "description": "This is access point group 4",
+            },
+            {
+                "name": "Access Point Group 5",
+                "controller": controllers[4].pk,
+            },
+            {
+                "name": "Access Point Group 6",
+            },
+        ]
+        cls.bulk_update_data = {
+            "controller": controllers[6].pk,
+            "tenant": tenants[6].pk,
+        }
+
+
+class SupportedDataRateTest(APIViewTestCases.APIViewTestCase):
+    model = models.SupportedDataRate
+    choices_fields = ["standard"]
+
+    @classmethod
+    def setUpTestData(cls):
+        models.SupportedDataRate.objects.create(rate=1000, standard="802.11b", mcs_index=1)
```

Comment:
Should we be using the appropriate ChoiceSet values rather than string literals for `standard` values?